ngraph/core/src/op/prelu.cpp

```diff
@@ -135,5 +140,6 @@ namespace prelu
 bool op::PRelu::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v0_PRelu_evaluate);
-    return prelu::evaluate_prelu(inputs[0], inputs[1], outputs[0]);
+    AutoBroadcastSpec broadcast_spec = AutoBroadcastSpec("NUMPY");
```

Comment:
As I know and see in your code the PReLU operation doesn't have and support broadcast.
Why did you add a broadcast support to the reference implementation? Here as I see you also always hardcode broadcast to "NUMPY".

Reply:
Extended, default is "NUMPY". Specified broadcast type can be passed in. As multiple inherit introduce lot of ambiguity, I incorporate broadcast function into op::PRelu directly. Thank you!

Reply:
@chenhu-wang We cannot extend existed nGraph operations. I propose to introduce the new version of PReLU operation with broadcast support if it is needed.
@lazarevevgeny Do you agree with me?

Reply:
@ilyachur is right. We cannot modify existing ngraph operations. We must add a new version of the operation and modify all the transformations which use v1::PRelu operation to work with a new version of the operation. Moreover, we need to modify the MO to emit a new version of the operation because right now it emits v1::PRelu.

Reply:
@ilyachur @lazarevevgeny ,I agree we should support backward compatibility, but from PRelu specification: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/PReLU_1.md, saying broadcasted capability is supported for this version. Should we modify the old one and add an new version spec for prelu?

Reply:
We should update the v1 specification that only the numpy style broadcasting is supported and create a new document for a v7 version of the PReLU (if we plan to introduce it in the upcoming release).

Reply:
this changes should be in other PR and reverted here, thank you.